ServerCore/Pages/Resources/nipd2023/HomePartial.cshtml

```diff
@@ -0,0 +1,54 @@
+﻿@{
+    var eventPage = Model as ServerCore.ModelBases.EventSpecificPageModel;
+    var Event = eventPage?.Event;
+}
+
+@if (!string.IsNullOrEmpty(Event.HomeContent))
+{
+    @Html.Raw(Event.HomeContent)
+}
+else
+{
+
+    <div class="container" id="pd2023-container">
+        <h2 class="pd2023-title">
+            <span>Non-Intern Puzzleday 2023</span>
+        </h2>
+
+        <div class="pd2023-container-root">
+            <div class="pd2023-container-card card-1 column-container">
+                <div class="column-one-third">
+                    <h3>WELCOME TO NIPD</h3>
+                    <p>Non-Intern Puzzleday is a fun 6-hour event where teams of up to 6 players solve puzzles. We accept puzzle solvers of all levels! Check out our <b><a asp-page="/EventSpecific/FAQ">FAQ</a></b> and <b><a asp-page="/EventSpecific/Rules">Rules</a></b> pages for more information.</p>
+                </div>
+                <div class="column-two-third">
+                    <img src="~/images/pd2023/nipd2023logo.png" height="160" width="160" title="Puzzleday 2023 logo" alt="Puzzleday 2023 logo" />
+                </div>
+             </div>
+
+            <div class="pd2023-container-card card-2">
+                <h3>NEW TO PUZZLES?</h3>
+                <p>You may have seen a jigsaw puzzle, a crossword puzzle, or a Sudoku at some point in your life, but those barely crack the cover on the whole anthology that is &quot;puzzling&quot;! Puzzleday puzzles also have a tendency to get... creative, so you might see something that looks especially strange! Check out a few <a asp-page="/EventSpecific/Samples">sample puzzles</a> or peruse our <a href="https://puzzlearchive.azurewebsites.net/puzzleday/main.html" target="_blank">collection of puzzles</a> from past events.</p>
+            </div>
+
+            <div class="pd2023-container-card card-3">
+                <h3>FULL SCHEDULE (all times PDT)</h3>
+                <p>This year's event will take place on <b>August 26<sup>th</sup></b>. Additionally, pregame puzzles will be available in the days leading up to the event.  On the day of:</p>
```

Comment:
Dana was working on this page in parallel, and I like her schedule more with the exception of your reminder that there will be live events in 99.

FULL SCHEDULE (all times PDT)
August 21 - 25 (optional pregame)

August 26 - DAY OF EVENT
10:30 Meet in 99 to sign in and get important information.
11:00 Opening ceremony.
11:10-5:00 Solve puzzles.
5:15-6ish Meet in 99 to celebrate the winners, see the solutions, and chat with the puzzle creators.

Reply:
Done!